server/rooms.ts

```diff
@@ -1422,9 +1422,11 @@ export class GlobalRoomState {
 		const upperStaffRoom = Rooms.get('upperstaff');
```

Comment:
I sadly can't suggest further up, but line 1416 should be:

```
const stack = err.stack || err.message || err.name || '';
const stackLines = (err ? Utils.escapeHTML(stack).split(`\n`) : []);
```

Also `@ts-ignore` is no longer necessary.